plugins/modules/vmware_cluster.py

```diff
@@ -328,9 +328,9 @@ def configure_ha(self):
         Returns: Cluster DAS configuration spec
 
         """
-        msg = 'Configuring HA using vmware_cluster module is deprecated and will be removed in version 2.12. ' \
+        msg = 'Configuring HA using vmware_cluster module is deprecated and will be removed in version 2.0.0. ' \
```

Comment:
We are currently at 1.0.0, and we plan to release every month. We follow semver ( https://semver.org/ ) and so we should bump the minor digit, so next release at the end of the month will be 1.1.0. And in 6 months with will be at 1.6.0. 2.12 is supposed to be released in 2022, and so I would go for a 1.24.0. What do you think?

Reply:
This was mapped to version decided in https://meetbot.fedoraproject.org/ansible-community/2020-06-10/community_working_group.2020-06-10-18.04.html

Reply:
Thanks for the link. So we bump the major after every Ansible core release, make sense since we will break our API by removing the  deprecated modules.